lib/common.sh

```diff
@@ -153,24 +158,32 @@ export IPAMPATH="${IPAMPATH:-${M3PATH}/ip-address-manager}"
 export IPAM_BASE_URL="${IPAM_BASE_URL:-metal3-io/ip-address-manager}"
 export IPAMREPO="${IPAMREPO:-https://github.com/${IPAM_BASE_URL}}"
 
-export IPA_DOWNLOAD_ENABLED="${IPA_DOWNLOAD_ENABLED:-true}"
-
 export CAPIPATH="${CAPIPATH:-${M3PATH}/cluster-api}"
 export CAPI_BASE_URL="${CAPI_BASE_URL:-kubernetes-sigs/cluster-api}"
 export CAPIREPO="${CAPIREPO:-https://github.com/${CAPI_BASE_URL}}"
 
-export BMOREPO="${BMOREPO:-https://github.com/metal3-io/baremetal-operator.git}"
-export BMO_BASE_URL="${BMO_BASE_URL:-metal3-io/baremetal-operator}"
-export FORCE_REPO_UPDATE="${FORCE_REPO_UPDATE:-true}"
 export BMOCOMMIT="${BMOCOMMIT:-HEAD}"
 export CAPM3COMMIT="${CAPM3COMMIT:-HEAD}"
 export IPAMCOMMIT="${IPAMCOMMIT:-HEAD}"
 export CAPICOMMIT="${CAPICOMMIT:-HEAD}"
 
-export BUILD_CAPM3_LOCALLY="${BUILD_CAPM3_LOCALLY=-false}"
-export BUILD_IPAM_LOCALLY="${BUILD_IPAM_LOCALLY=-false}"
-export BUILD_BMO_LOCALLY="${BUILD_BMO_LOCALLY=-false}"
-export BUILD_CAPI_LOCALLY="${BUILD_CAPI_LOCALLY=-false}"
+export IRONIC_IMAGE_PATH="${IRONIC_IMAGE_PATH:-/tmp/ironic-image}"
+export IRONIC_IMAGE_REPO="${IRONIC_IMAGE_REPO:-https://github.com/metal3-io/ironic-image.git}"
+export IRONIC_IMAGE_BRANCH="${IRONIC_IMAGE_BRANCH:-main}"
+export IRONIC_IMAGE_COMMIT="${IRONIC_IMAGE_COMMIT:-HEAD}"
+
+export BUILD_CAPM3_LOCALLY="${BUILD_CAPM3_LOCALLY:-false}"
+export BUILD_IPAM_LOCALLY="${BUILD_IPAM_LOCALLY:-false}"
+export BUILD_BMO_LOCALLY="${BUILD_BMO_LOCALLY:-false}"
+export BUILD_CAPI_LOCALLY="${BUILD_CAPI_LOCALLY:-false}"
+export BUILD_IRONIC_IMAGE_LOCALLY="${BUILD_IRONIC_IMAGE_LOCALLY:-false}"
```

Comment:
Is there any reason to add` _IMAGE` for ironic only and not for other components?

Reply:
Well nothing super special, only that the repo name is ironic-image. I was also debating whether it should be just IRONIC but Ironic is written in python so the application itself is never built strictly speaking. But I am fine with just IRONIC if you would like to change to that.

Reply:
Not a big deal.
/approve